src/witan/models/demography/trend_based_ccm/fertility/historic_asfr_births_by_age_of_mother.clj

```diff
@@ -1,5 +1,34 @@
-(ns witan.models.demography.trend-based-ccm.fertility.historic-asfr-births-by-age-of-mother)
+(ns witan.models.demography.trend-based-ccm.fertility.historic-asfr-births-by-age-of-mother
+  (:require [clojure.java.io :as io]
+            [clojure.string :as str]
+            [clojure.walk :as walk]
+            [clojure-csv.core :as csv]
+            [clojure.core.matrix.dataset :as ds]))
 
+;; LOADING DATA - used for development purposed - to be removed
+(defn- load-csv
```

Comment:
This could be done from the test namespace instead. Happy to discuss how to do it.